### ISSUE TITLE
Hierarchical facet is not keyboard accessible

### ISSUE THREAD

**Issue by mhk33** [sul-dlss/blacklight-hierarchy#18]
It appears that you can tab to the top-level category, but that category cannot be expanded using keyboard only.
